Addresses comments in issue 139 (#140)
cisagov/ScubaGoggles: baselines/Gmail Minimum Viable Secure Configuration Baseline v0.1.md

## 8. User Email Uploads

> This section deals with the feature that allows users to upload emails.

Reviewer: ```suggestion
This section deals with the feature that enables users to import their email and contacts from non-Google webmail accounts such as Yahoo!, Hotmail, or AOL.
```

> This section deals with the feature that enables users to import their email and contacts from non-Google webmail accounts such as Yahoo!, Hotmail, or AOL.

Reviewer 2: One minor comment that might just be personal preference. "The feature" implies that this is the one and only way for users to do this. I'd prefer "a feature" instead of "the feature" as it's more open ended. 
```suggestion
This section addresses a feature that enables users to import their email and contacts from non-Google webmail accounts such as Yahoo!, Hotmail, or AOL.
```
